### PR TITLE
content and accessibility changes

### PR DESCRIPTION
**Pull request by stephenjmcneill1971**
- Changed text 'open in new window' to 'open in new tab'
- Added 'select all that apply' to relevant pages
- Changed second postcode hint test in address pages
- Added address line text to all inputs address pages

### CODE REVIEW

File: src/lib/resource/sections/context-pregnancy.js [hunk 1]
```diff
@@ -18,9 +18,9 @@ module.exports = {
...
...more evidence from you.</p><h2 class="govuk-heading-m">Help and support</h2><p class="govuk-body">You can get practical and emotional support after a crime. Visit <a class="govuk-link" href="https://www.victimandwitnessinformation.org.uk/" target="_blank">Victim and Witness information (opens in new window)</a> if you’re in England and Wales. Or <a class="govuk-link" href="https://www.mygov.scot/victim-witness-support/" target="_blank">support as a victim or witness of crime (opens in new window)</a> in Scotland.</p>{% set templateHtml %}{% include \'contact.njk\' %}{% endset %}{{ govukDetails({summaryText: "If you need help or support with the application",html: \'<p class="govuk-body">You can contact us for help with your application.</p>\' + templateHtml + \'<p class="govuk-body">You can <a class="govuk-link" href="https://www.victimandwitnessinformation.org.uk/" target="_blank">get practical or emotional support (opens in new window)</a> after a crime.</p><p class="govuk-body">There is different practical or emotional support <a class="govuk-link" href="https://www.mygov.scot/victim-witness-support/" target="_blank">if you live in Scotland (opens in new window)</a>.</p>\'})}}',
```
Comment (tjbburton) [inline src/lib/resource/sections/context-pregnancy.js:21]: Should this say new tab instead of new window?

**Review by tjbburton — CHANGES_REQUESTED** [review]
Double check whether links open in new tab or window

File: src/lib/resource/sections/context-pregnancy.js (hunk 1 above)
Comment (stephenjmcneill1971) [reply src/lib/resource/sections/context-pregnancy.js:21]: The ticket has been amended the "tabs" is correct

File: src/lib/resource/sections/context-pregnancy.js [hunk 2]
```diff
@@ -18,9 +18,9 @@ module.exports = {
...
... more evidence from you.</p><h2 class="govuk-heading-m">Help and support</h2><p class="govuk-body">You can get practical and emotional support after a crime. Visit <a class="govuk-link" href="https://www.victimandwitnessinformation.org.uk/" target="_blank">Victim and Witness information (opens in new window)</a> if you’re in England and Wales. Or <a class="govuk-link" href="https://www.mygov.scot/victim-witness-support/" target="_blank">support as a victim or witness of crime (opens in new window)</a> in Scotland.</p>{% set templateHtml %}{% include \'contact.njk\' %}{% endset %}{{ govukDetails({summaryText: "If you need help or support with the application",html: \'<p class="govuk-body">You can contact us for help with your application.</p>\' + templateHtml + \'<p class="govuk-body">You can <a class="govuk-link" href="https://www.victimandwitnessinformation.org.uk/" target="_blank">get practical or emotional support (opens in new window)</a> after a crime.</p><p class="govuk-body">There is different practical or emotional support <a class="govuk-link" href="https://www.mygov.scot/victim-witness-support/" target="_blank">if you live in Scotland (opens in new window)</a>.</p>\'})}}'
```
[the remark this answers is not in the document]
Comment (stephenjmcneill1971) [reply src/lib/resource/sections/context-pregnancy.js:23]: The ticket has been amended the "tabs" is correct

**Review by tjbburton — APPROVED** [review]

**Review by tjbburton — APPROVED** [review]

File: src/lib/resource/sections/applicant-dentist-address.js [hunk 3]
```diff
@@ -89,7 +89,7 @@ module.exports = {
                             properties: {
                                 'q-applicant-dentist-address-building-and-street-3': {
                                     type: 'string',
-                                    title: 'Building and street line 3',
+                                    title: 'Address line 2 (optional)',
```
Comment (tjbburton) [inline src/lib/resource/sections/applicant-dentist-address.js:92]: Should be address line 3

**Review by tjbburton — CHANGES_REQUESTED** [review]
applicant-dentist-address has the wrong address line label for line 3

File: src/lib/resource/sections/applicant-dentist-address.js (hunk 3 above)
Comment (stephenjmcneill1971) [reply src/lib/resource/sections/applicant-dentist-address.js]: Changed

**Review by tjbburton — APPROVED** [review]

**Review by tjbburton — APPROVED** [review]

**Pull request merged**